test/modules/otjson-importer.test.js

```diff
@@ -1,330 +0,0 @@
-const OtJsonImporter = require('../../modules/importer/ot-json-importer');
```

Comment:
Since we're deleting otjson-importer.test, does it make sense to add importer-service.test?